package.json

```diff
@@ -17,28 +17,31 @@
+        "pretest": "npm run build:schemas",
+        "test": "jest -c jest.unit.config.js",
+        "test:coverage": "npm run test -- --coverage",
         "build:schemas": "node scripts/generate-schema-ts.js",
-        "build:webpack:dev": "npm run build:schemas && webpack --config webpack/webpack.dev.js",
-        "build:webpack:prod": "npm run build:schemas && webpack --config webpack/webpack.prod.js",
-        "build:es": "npm run build:schemas && tsc --project ./tsconfig.es.json",
-        "build:cjs": "npm run build:schemas && tsc --project ./tsconfig.cjs.json",
+        "build:webpack:dev": "webpack --config webpack/webpack.dev.js",
+        "build:webpack:prod": "webpack --config webpack/webpack.prod.js",
+        "build:dist:es": "tsc --project ./tsconfig.es.json",
+        "build:dist:cjs": "tsc --project ./tsconfig.cjs.json",
+        "build:assets": "npm-run-all -p build:dist:* build:webpack:prod",
+        "build": "npm-run-all -s build:schemas build:assets",
+        "build:dev": "npm-run-all -s build build:webpack:dev",
+        "release": "npm run build",
         "version": "standard-version",
         "lint": "npx tslint --config ./tslint.json --project ./tsconfig.webpack.json --format verbose",
         "lint:fix": "npx tslint -c ./tslint.json -p ./tsconfig.webpack.json --format verbose --fix",
         "prettier": "npx prettier --check .",
         "prettier:fix": "npx prettier --write .",
         "prepublishOnly": "npm run release",
-        "test:coverage": "npm run build:webpack:dev && jest -c jest.unit.config.js --coverage",
-        "integ:local:chrome": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe chrome --app \"http-server ./build/dev -s\"",
-        "integ:local:chrome:headless": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe chrome:headless --app \"http-server ./build/dev -s\"",
-        "integ:local:firefox": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe firefox --app \"http-server ./build/dev -s\"",
-        "integ:local:firefox:headless": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe firefox:headless --app \"http-server ./build/dev -s\"",
-        "integ:local:safari": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe safari --app \"http-server ./build/dev -s\"",
-        "integ:local:edge": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe edge --app \"http-server ./build/dev -s\"",
-        "integ:local:ie": "npm run build:webpack:dev && cross-env NODE_OPTIONS=--max-http-header-size=80000 testcafe ie --app \"http-server ./build/dev -s\" src/__integ__/unsupportedBrowser.test.ts",
+        "preinteg:browser": "npm run build:dev",
+        "integ:browser": "testcafe --config-file=./.testcafe.browser.js --app \"http-server ./build/dev -s\"",
```

Comment:
issue: The nice thing about having each browser as a separate script is that it is convenient for development. We could run testcafe directly, but it's convenient to simply call `npm run integ:local:chrome`.

Reply:
forked from there, bit there is nothing preventing you from doing:

```npm run integ:browser -- chrome```

which will run tests in chrome :-)

Reply:
oh I see -- yep that seems like an elegant solution